Lesson1_business-card/index.html

```diff
@@ -17,27 +17,27 @@
         <div class="personal-data">
           <div class="name-data">
             <div class="name-surname">
-              <div class="text name">Aleksander</div>
-              <div class="text surname">KOWALSKI</div>
+              <div class="name">Aleksander</div>
+              <div class="surname">KOWALSKI</div>
             </div>
-            <div class="text work">STANOWISKO</div>
+            <div class="work">STANOWISKO</div>
           </div>
-          <div class="text contact-data">
+          <div class="contact-data">
```

Comment:
remember about indentation the img should be at the same lvl as the span

Reply:
fixed